### PR TITLE
Liquid jet

### PR DESCRIPTION
**Pull request by vespos**
## Description
Add liquid_jet class for MFX and XCS liquid jet setups.

## Motivation and Context
Devices used regularly at MFX, XCS, and XPP.

## How Has This Been Tested?
Used at MFX to generate the screens based on the happi entry pointing to this class.

## Where Has This Been Documented?
Docstrings

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [ ] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: pcdsdevices/liquid_jet.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class LiquidJet(BaseInterface, GroupDevice):
+    """
+    Liquid jet motion class.
+
+    This class controls the axis for the liquid jet setups
+    """
+    tab_component_names = True
+    #_icon = 'fa.minus-square'
```
Comment (tangkong) [inline pcdsdevices/liquid_jet.py:15]:
```suggestion
    # _icon = 'fa.minus-square'
```
Style dictates we put a space after the comment '#'.  Did you want this to have an icon though?

File: pcdsdevices/liquid_jet.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    #_icon = 'fa.clock-o'
```
Comment (tangkong) [inline pcdsdevices/liquid_jet.py:16]:
```suggestion
    # _icon = 'fa.clock-o'
```
same as above.  Though if we have one we should delete the other

**Review by tangkong — CHANGES_REQUESTED** [review]
The test suite failed on a test that's seemingly unrelated to this.  looking into it now, but in the meantime there's a couple quick style changes that'd be nice to get in.

**Comment by ZLLentz** [comment]
reference to the flaky test issue, must be unrelated to this PR: https://github.com/pcdshub/pcdsdevices/issues/1050

**Comment by ZLLentz** [comment]
I think we can merge if you uncomment, remove, or fix the style thing on the lines that Robert pointed out above.

Currently this icon string is used in lightpath to pick an icon.

**Comment by vespos** [comment]
Hi thanks both for your feedback. I actually realized that we already have a jet class (jet.py). I'd have to check that it works fine with the newer version of the jet that we have at MFX. Or we might consider updating the old XCS liquid jet to a newer version compatible with the current Beckhoff axis class.

**Comment by tangkong** [comment]
Ah!  You're absolutely right.  It looks like the `pcdsdevices.jet.BeckhoffJet` class should reproduce what you've written in this PR, unless the `BeckhoffAxis_Pre140` is out of date.  A quick look through happi seems to suggest that there aren't any devices using `BeckhoffJet` right now, so updating the `BeckhoffAxis_Pre140` components could be an un-disruptive option.

**Pull request closed without merging**